src/main/java/Winter.java

```diff
@@ -0,0 +1,161 @@
+        String line = "-----------------------------------\n";
+        String indent = "   ";
+        String echoLine;
+        Scanner input = new Scanner(System.in);
+        echoLine = input.nextLine();
+        while (true) {
+            if(echoLine.equals("bye") || echoLine.equals("Bye") || echoLine.equals("BYE")) {
+                break;
+            }
+            System.out.print(line);
+            System.out.print(indent);
+            System.out.println(echoLine);
+            System.out.print(line);
+            echoLine = input.nextLine();
+        }
+
+    }
+    // Method for adding tasks by handling user commands
+    private static void addTasks() {
+        String line = "-----------------------------------\n";
+        String indent = "   ";
+        Task[] taskList = new Task[100];
+        int taskIndex = 0;
+        Scanner input = new Scanner(System.in);
+        while (true) {
+            String command = input.nextLine();
+            Task newTask = new Task(taskIndex+1,false,command);
+            taskList[taskIndex] = newTask;
+
+            switch(command) {
+            // Cases include farewell and list commands
+            case "bye":
+                break;
+            case "Bye":
+                break;
+            case "BYE":
+                break;
+            case "list":
+                displayList(taskList,taskIndex);
+                continue;
+            case "List":
+                displayList(taskList,taskIndex);
+                continue;
+            case "LIST":
+                displayList(taskList,taskIndex);
+                continue;
+
+            }
+            switch(command.substring(0,command.indexOf(" "))) {
+            // Cases for marking tasks
+            case "mark":
+                taskList[Integer.parseInt(command.substring(5,command.length()))-1].mark();
+                continue;
+            case "unmark":
+                taskList[Integer.parseInt(command.substring(7,command.length()))-1].unmark();
+                continue;
+            }
+            taskIndex++;
+            System.out.print(line);
+            System.out.print(indent);
+            System.out.println("added: " + command);
+            System.out.print(line);
+
+        }
+    }
+    // Method for displaying list
+    private static void displayList(Task[] taskList, int taskIndex) {
+        String line = "-----------------------------------\n";
+        String indent = "   ";
+        for (int i = 0; i < taskIndex; i++) {
+            String status;
+            if (taskList[i].isMarked()) {
+                status = "[X]";
+            } else {
+                status = "[ ]";
+            }
+            System.out.print(indent);
+            System.out.println(taskList[i].getOrder() + ". " + status + " " + taskList[i].getTaskName());
+        }
+        System.out.print(line);
+    }
+    // Class for Task object
+    private static class Task {
+        boolean marked;
```

Comment:
This boolean should be named to sound like booleans. Consider changing it to "isMarked"?